src/open_inwoner/js/components/upload-document/show-file-info.js

```diff
@@ -1,95 +1,111 @@
-const fileUploadInput = document.getElementById('id_file')
-const sizeInfo = document.getElementById('upload_size')
-const nameInfo = document.getElementById('upload_name')
-const validationInfo = document.getElementById('upload_error')
-const closeButton = document.getElementById('close_upload')
-const submit_upload = document.getElementById('submit_upload')
-// show/hide after validation
-const iconDrive = document.querySelectorAll('.drive')
-// show info
-const formControlInfo = document.querySelectorAll('.form__control--info')
+export class ShowInfo {
+  static selector = '.upload'
 
-if (fileUploadInput) {
-  // Convert the file size to a readable format
-  const formatFileSize = function (bytes) {
-    const suffixes = ['B', 'kB', 'MB', 'GB', 'TB']
-    const i = Math.floor(Math.log(bytes) / Math.log(1024))
-    return `${(bytes / Math.pow(1024, i)).toFixed(2)} ${suffixes[i]}`
+  constructor(node) {
+    this.fileUploadInput = node
+    this.showData()
   }
-  // initial values
-  submit_upload.disabled = true
-  validationInfo.classList.add('error')
-  closeButton.classList.add('error')
-  iconDrive.forEach((elem) => {
-    elem.classList.add('error')
-  })
 
-  formControlInfo.forEach((elem) => {
-    elem.style.display = 'none'
-  })
+  showData() {
+    // get the closest parent element (the form section element)
+    const documentUpload = this.fileUploadInput.closest('#form_upload')
 
-  fileUploadInput.addEventListener('change', function (e) {
-    const files = e.target.files
+    const sizeInfo = documentUpload.querySelector('#upload_size')
+    const nameInfo = documentUpload.querySelector('#upload_name')
+    const validationInfo = documentUpload.querySelector('#upload_error')
+    const closeButton = documentUpload.querySelector('#close_upload')
+    const submit_upload = documentUpload.querySelector('#submit_upload')
```

Comment:
This still uses ID's, eg: global selectors. I'm not sure what browsers do with duplicate ID's even if you'd `querySelector from a one of their ancestor elements, or how this works with closest. 

The point of this is making sure that when hydrating one fragment we're not accidentally select elements elsewhere in the page. 

In this case I'll not block on this as there won't be multiple of this element.